### ISSUE TITLE
Set up issue

### ISSUE THREAD

**Issue by gcwsilver** [jomariya23156/sales-forecast-mlops-at-scale#3]
I have some problems setting up and I do not know how to solve them. 
I am setting up with docker-compose on Windows. The first thing that makes me confused is whether I should type wsl first or not. In this case, I did not do it but the program still seems to work.
![messageImage_1713779524941](https://github.com/jomariya23156/sales-forecast-mlops-at-scale/assets/67415825/5ce4aa44-938d-4460-9cc7-fb7e6edd06ee)
After installation, there is an error from the daemon path and it seems like none of the UI in the container works. Or am I missing a way to open other programs?
![image (6)](https://github.com/jomariya23156/sales-forecast-mlops-at-scale/assets/67415825/fd53355b-4bd6-4d01-847f-e124c065ed62)

**Comment by gcwsilver** [comment]
Another error I received while installing is this error
![image (5)](https://github.com/jomariya23156/sales-forecast-mlops-at-scale/assets/67415825/13a5a77a-3936-488e-bcd0-9d2d367cae39)

**Comment by jomariya23156** [comment]
I haven't used WSL and Docker on Windows for a while now, but I highly recommend sticking with WSL for local development on Windows. It's more Linux-native and will make your developer journey much easier. As I recall, I installed Docker Desktop normally, and then there's a setting to integrate Docker into WSL.

From your first screenshot, it seems like Docker is working fine on your WSL. Sometimes the setup can go wrong or even freeze because the system is resource-consuming and your computer might be struggling to manage the required resources. In my experience, simply rerunning the command or waiting a bit longer (even if the status shows 'unhealthy') often resolves the issue, and Kafka and Ray start up correctly.

To investigate this further, try starting the system without Airflow first. This will make the setup faster and less resource-consuming. Your setup command would then be simplified to `docker-compose up`. Afterward, examine the logs of all services for any warnings or errors.

For reference, during development, I allocated 11GB of RAM and 6 CPU cores to Docker. It worked fine most of the time, but occasionally I encountered startup errors like yours. Simply rerunning the command usually solved the issue.
